roles/utilities/tasks/swap.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: check for swap
+  stat: path=/var/swap.1
+  register: swap
+
+- name: allocate swap
+  command: /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=1024
```

Comment:
sort-of ontopic: probably should avoid using a ssd for this considering the amount of data it might consume, especially when importing logs